### PR TITLE
Clean sphinx config

### PR DESCRIPTION
**Pull request by ssbarnea**
- removes invalid objects.inv entries
- reformatted by black on save
- some other minor touches

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -37,17 +36,17 @@
...
 
 # The master toctree document.
-master_doc = 'index'
+master_doc = "index"
 
 # General information about the project.
 project = "Molecule"
-copyright = " %s, Red Hat Inc." % datetime.date.today().year
+copyright = f" {datetime.date.today().year}, Red Hat Inc."
 author = "AUTHORS.rst"
```
Comment (ziegenberg) [inline docs/conf.py:50]: The file `AUTHORS.rst` does not exist.

File: docs/conf.py [hunk 2]
```diff
@@ -278,33 +277,17 @@
...
     "ansible": ("https://docs.ansible.com/ansible/latest/", None),
+    "jinja2": ("http://jinja.palletsprojects.com/", None),
     "pip": ("https://pip.pypa.io/en/latest/", None),
     "python": ("https://docs.python.org/3", None),
-    "python2": ("https://docs.python.org/2", None),
     "testinfra": ("https://testinfra.readthedocs.io/en/latest/", None),
-    "yamllint": ("http://yamllint.readthedocs.io/en/latest/", None),
     "virtualenv": ("https://virtualenv.pypa.io/en/latest/", None),
-    }
+    "yamllint": ("http://yamllint.readthedocs.io/en/latest/", None),
```
Comment (ziegenberg) [inline docs/conf.py:289]:
```suggestion
    "yamllint": ("https://yamllint.readthedocs.io/en/latest/", None),
```

the build complains:
```
intersphinx inventory has moved: http://yamllint.readthedocs.io/en/latest/objects.inv -> https://yamllint.readthedocs.io/en/latest/objects.inv
```

File: docs/conf.py [hunk 3]
```diff
@@ -278,33 +277,17 @@
...
-                       'python3': ('https://docs.python.org/3/', (None, '../python3.inv')),
-                       'jinja2': ('http://jinja.palletsprojects.com/', (None, '../jinja2.inv')),
-                       'ansible_2_9': ('https://docs.ansible.com/ansible/2.9/', (None, '../ansible_2_9.inv')),
-                       'ansible_2_8': ('https://docs.ansible.com/ansible/2.8/', (None, '../ansible_2_8.inv')),
-                       'ansible_2_7': ('https://docs.ansible.com/ansible/2.7/', (None, '../ansible_2_7.inv')),
-                       'ansible_2_6': ('https://docs.ansible.com/ansible/2.6/', (None, '../ansible_2_6.inv')),
-                       'ansible_2_5': ('https://docs.ansible.com/ansible/2.5/', (None, '../ansible_2_5.inv')),
+intersphinx_mapping = {
     "ansible": ("https://docs.ansible.com/ansible/latest/", None),
+    "jinja2": ("http://jinja.palletsprojects.com/", None),
```
Comment (ziegenberg) [inline docs/conf.py:284]:
```suggestion
    "jinja2": ("https://jinja.palletsprojects.com/", None),
```
the build complains:
```
intersphinx inventory has moved: http://jinja.palletsprojects.com/objects.inv -> https://jinja.palletsprojects.com/en/3.1.x/objects.inv
```

Comment (ssbarnea) [reply docs/conf.py]: I am not planning to use the non-permalink version.

File: docs/conf.py (hunk 1 above)
Comment (ssbarnea) [reply docs/conf.py]: Fixed

**Review by ziegenberg — APPROVED** [review]

**Pull request merged**